docs/Makefile

```diff
@@ -46,6 +46,7 @@ apidocs:
 	rsync -a ../sdk/java-sdk-testkit/target/api/ "${managed_attachments}/testkit/"
 	rsync -a ../sdk/scala-sdk/target/scala-2.13/api/ "${managed_attachments}/scala-api/"
 	rsync -a ../sdk/scala-sdk-testkit/target/scala-2.13/api/ "${managed_attachments}/scala-testkit-api/"
+	bin/version.sh | xargs -0  printf "%s" > "${managed_attachments}/latest-version.txt"
```

Comment:
do you still have to commit `bin/version.sh`?

in `/updatePomVersions.sh` we do `SDK_VERSION=$(sbt "print sdkJava/version" | tail -1)`

Reply:
oh, `bin/version.sh` was there already. Pretty handy :)